### PR TITLE
Feature/proxy 12

### PR DESCRIPTION
**Pull request by kkorchyts**

### CODE REVIEW

**Review by m-bondarev — APPROVED** [review]

File: src/api/app.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import express from "express"
-import {nasaAsteroidsHandler} from "./handler/nasa-asteroids.handler.js";
-import {checkQueryParamMiddleware} from "./middleware/nasa-asteroids.middleware.js";
+import {handler as nasaAsteroidsHandler} from "./handler/nasa-asteroids.handler.js";
```
Comment (plutsenka) [inline src/api/app.js:2]: Don't mix up `''` and `""` in your code, please

**Review by plutsenka — APPROVED** [review]

File: src/api/app.js (hunk 1 above)
Comment (mariaYurcheuskaya) [reply src/api/app.js:2]: you can use prettier for it)))

File: src/api/schemas/date.schema.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Joi from 'joi';
+import JoiDate from "@joi/date";
+
+const extendedJoi = Joi.extend(JoiDate);
+
+export const dateSchema = extendedJoi.date().required()
+    .format('YYYY-MM-DD')
```
Comment (mariaYurcheuskaya) [inline src/api/schemas/date.schema.js:7]: How do you think about making it a constant ?

File: src/api/app.js (hunk 1 above)
Comment (kkorchyts) [reply src/api/app.js:2]: I'll improve in the next PR

File: src/api/schemas/date.schema.js (hunk 2 above)
Comment (kkorchyts) [reply src/api/schemas/date.schema.js:7]: I'll move it into constant if it appears more in one place. I propose leaving it as is. Could you, please, share your thoughts with me?

Comment (mariaYurcheuskaya) [reply src/api/schemas/date.schema.js:7]: yep, agree

**Review by mariaYurcheuskaya — APPROVED** [review]

**Pull request merged**